Formula/apib.rb

```diff
@@ -14,28 +13,21 @@ class Apib < Formula
     sha256 "bbe9bc25a8584f163347662675d78b69cdfaac495be5f2fa026dfca112f8d4a4" => :sierra
   end
 
-  depends_on "apr"
-  depends_on "apr-util"
+  depends_on "cmake" => :build
+  depends_on "libev"
   depends_on "openssl@1.1"
 
   def install
-    # Fix detection of libcrypto for OpenSSL 1.1
-    inreplace "configure", "CRYPTO_num_locks", "EVP_sha1"
+    ENV.append "EXTRA_INCLUDE_DIR", "${prefix}/include"
+    ENV.append "EXTRA_LIB_DIR", "${prefix}/lib"
```

Comment:
`prefix` in this context actually is set to "/usr/local/Cellar/apib/1.2.1`. Again, I suspect it is not needed here - Homebrew automatically adds the include and lib directories of the dependencies.